lms/templates/footer.html

```diff
@@ -2,42 +2,85 @@
 <%!
   from django.utils.translation import gettext as _
   from lms.djangoapps.branding.api import get_footer
+  from django.conf import settings
+  from datetime import datetime
+%>
+<%
+  footer = get_footer(is_secure=request.is_secure())
+  footer['copyright'] = _(u"\u00A9 {year} Massachusetts Institute of Technology").format(year=datetime.now().year)
```

Comment:
This should be the same as in MFE:

<img width="1792" height="179" alt="Image" src="https://github.com/user-attachments/assets/e8b5ff3d-af83-4d83-be01-5f38bee2233b" />

```suggestion
  footer['copyright'] = _(u"\u00A9 {year} MITx Residential. All rights reserved.").format(year=datetime.now().year)
```